app/template/create/page.tsx

```diff
@@ -2,27 +2,54 @@
 
 import React from 'react';
 import {
+  Breadcrumb,
+  Breadcrumbs,
   Button,
   FieldError,
   Form,
   Input,
   Label,
+  Link,
   Radio,
   RadioGroup,
   Text,
   TextField
 } from "react-aria-components";
+import PageHeader from "@/components/PageHeader";
 
 const TemplateCreatePage: React.FC = () => {
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+    console.log('Form submitted');
+
+    // redirect to the new template page
+     window.location.href = '/template/create/select-template';
+
+  }
+
   return (
     <>
-      <h1>Create a template</h1>
 
-      <p>
 
-      </p>
+      <PageHeader
```

Comment:
I'm wondering if we can standardize some spacing. For example, in this page, you can see that padding is needed above the button, and that there should be more consistent spacing above and below the "Template name" field row: 
<img width="787" alt="image" src="https://github.com/user-attachments/assets/5e7b212a-5f03-448e-b099-c489abf8f0f7">

Reply:
This page is a bit of a WIP - I will adjust it in a later separate PR

Reply:
Ok, thanks. Is there a ticket for this template? Could you share the link here, and I can update it with my comments.

Reply:
thank you